game/dota_addons/dota_imba/scripts/npc/npc_items_custom.txt

```diff
@@ -14138,6 +14138,31 @@
 				"var_type"				"FIELD_FLOAT"
 				"stun_duration"			"2"
 			}
+			"13"
+			{
+			"var_type"						"FIELD_INTEGER"
+				"damage_reduction"		"6"
```

Comment:
Oh, uhh, maybe it didn't have %DR before.  And maybe it doesn't after.  I wrote this code a few days ago and didn't want to forget about it.  \*shrug\*